MapleServer2/Types/Inventory.cs

```diff
@@ -52,6 +61,34 @@ public Inventory(IEnumerable<Item> loadItems) : this()
             }
         }
 
+        public Inventory(Inventory inventory) : this()
+        {
+            Equips = new Dictionary<ItemSlot, Item>();
+            Cosmetics = new Dictionary<ItemSlot, Item>();
+            Badges = new List<Item>();
+            Id = inventory.Id;
+            ExtraSize = inventory.ExtraSize;
+            foreach (Item item in inventory.DB_Items)
+            {
+                if (item.IsEquipped)
+                {
+                    Equips.Add(item.ItemSlot, item);
+                }
+                else if (item.IsEquipped && item.InventoryTab == InventoryTab.Outfit)
```

Comment:
Don't need to keep checking if the item is equipped on 77 and 81 because if it is, it will be true on line 73